docs:#1284 edit index pages (#1325)
redpanda-data/documentation: docs/manage/console/index.mdx

- [Kafka Connect](../console/kafka-connect)

    Redpanda Console provides a user interface to manage multiple Kafka connect clusters.

- [Schema Registry](../console/schema-registry)

>     To deserialize Kafka records that have been encoded with a schema registry compatible serializer you must have a valid schema registry configuration in Redpanda Console. 

Reviewer: ```suggestion
    Configure Redpanda Console to deserialize Kafka records that were encoded by a Schema Registry compatible serializer.
```
